### PR TITLE
refactor(i18n): localize Settings → General → Terminal

### PR DESCRIPTION
**Pull request by jcfs**
Settings → General → Terminal was still holding English literals, so it could not follow a locale switch. Its copy now resolves through i18next at render, and the page joins the `i18nGuardFiles` allowlist so it cannot drift back.

## Important Changes

- Shell paths, font family names, and every select `value` (`auto`, `custom`, `__custom__`, `new_tab`, `browser_panel`, `default`) are left untranslated — they are compared with `===` or submitted as data, and a translated sentinel stops matching in any non-English locale.
- `CATEGORY_LABELS` / `CATEGORY_BADGES` are module-scope tables, so they now hold catalog **keys** resolved at render (`CATEGORY_LABEL_KEYS` / `CATEGORY_BADGE_KEYS`). A `t()` there would freeze at the boot locale, and neither the lint guard nor the pseudo-locale can see that failure.
- Catalog additions are alphabetical in `src/locales/en/settings.json`; `Shell` and `System` reuse the existing `common` entries.

## Validation

```
pnpm exec tsc --noEmit      # clean
pnpm lint --max-warnings 0  # clean
pnpm run i18n:check         # 4/4 gates pass
pnpm exec vitest run        # 1016 files, 7769 passed
```

Guard confirmed live on the new paths: reintroducing one literal in `terminal-settings.tsx` produces `i18next/no-literal-string`, and reverting it goes clean again.

Pseudo-locale completeness check: rendered the whole `TerminalSettings` tree under `pseudo` with both selects open (16 options mounted) and swept every text node, `placeholder`, `aria-label` and `title` for plain ASCII. The only un-accented text is a Radix-injected `<style>` element; all copy renders accented, and the font/shell data values stay verbatim as intended.

The screen is also added to `SCREENS` in `e2e/tests/i18n/pseudo-coverage.spec.ts` (gated on `KANDEV_I18N_COVERAGE=1`). That spec walks visible text nodes only and never opens a select, so backend shell-option labels are not in the DOM during the crawl and need no `ALLOWED` entries.

New tests: `shell-settings-card.test.tsx` (copy renders from the catalog; the `/bin/zsh` example placeholder stays untranslated in the custom branch) and a `terminal-settings.test.ts` suite asserting the category tables hold keys, not copy, and that every `FontCategory` resolves.

## Possible Improvements

Low risk — copy-only, no behavior change. The shared settings chrome (`settings-card`, `settings-section`, `settings-save-provider`) is deliberately untouched; it belongs to a sibling migration PR.

## Checklist

- [ ] I have performed a self-review of my code.
- [ ] I have manually tested my changes and they work as expected.
- [ ] My changes have tests that cover the new functionality and edge cases.
- [ ] If my change touches UI files (`apps/web/`), I have added or updated Playwright e2e tests in `apps/web/e2e/` and verified them with `make test-e2e`.
- [ ] I checked whether this affects public docs in `docs/public/**` and updated them or noted why no docs change is needed.

### CODE REVIEW

File: apps/web/eslint.i18n.options.mjs [hunk 1]
```diff
@@ -159,12 +159,19 @@ export const i18nGuardFiles = [
   // Settings → General → Appearance, migrated end-to-end as the worked example:
   // the page, the two sections it owns, and the settings chrome around them.
   "app/settings/general/appearance/**/*.{ts,tsx}",
+  // Settings → General → Terminal: the page plus the two cards it owns. The
+  // shared settings chrome (settings-card, settings-section,
+  // settings-save-provider) is deliberately absent — it is migrated with the
+  // page that owns it.
+  "app/settings/general/terminal/**/*.{ts,tsx}",
```
[the remark this answers is not in the document]
Comment (jcfs) [reply apps/web/eslint.i18n.options.mjs:179]: You're right, and my stated reason for skipping it was wrong. I'd assumed the backend shell-option labels would render as plain-English leftovers, but `findUnlocalizedText` walks visible text nodes only — it never opens a select, so those labels are never in the DOM during the crawl. In the default state `preferredShell` is empty, so the trigger shows "System default" and the font trigger shows "Default (Menlo / Monaco)", both accented.

Added in dc6f9cb — `{ name: "settings — terminal", url: "/settings/general/terminal" }` alongside appearance, no ALLOWED additions needed. Correcting the claim in the PR description too.

File: apps/web/src/locales/pseudo/settings.json [hunk 2]
```diff
@@ -1,31 +1,40 @@
...
   "changesPanelLayout": "Ćĥàńĝēś Ƥàńēĺ Ĺàŷōũţ",
   "changesTheLanguageOfTheKandev": "Ćĥàńĝēś ţĥē ĺàńĝũàĝē ōƒ ţĥē Ķàńďēv ĩńţēŕƒàćē. Àƥƥĺĩēś ĩḿḿēďĩàţēĺŷ àńď ĩś ŕēḿēḿƀēŕēď ōń ţĥĩś ďēvĩćē. Ţàśķ ţĩţĺēś, ćĥàţ, àńď ōţĥēŕ ćōńţēńţ ŷōũ ćŕēàţē àŕē ńōţ ţŕàńśĺàţēď.",
   "chatInput": "Ćĥàţ Ĩńƥũţ",
   "chatInputAndCommandShortcuts": "Ćĥàţ ĩńƥũţ àńď ćōḿḿàńď śĥōŕţćũţś",
+  "chooseAMonospaceFontForThe": "Ćĥōōśē à ḿōńōśƥàćē ƒōńţ ƒōŕ ţĥē ţēŕḿĩńàĺ. Ńēŕď Ƒōńţś ĩńćĺũďē ĩćōńś ƒōŕ ĆĹĨ ţōōĺś.",
   "chooseTheLanguageUsedAcrossThe": "Ćĥōōśē ţĥē ĺàńĝũàĝē ũśēď àćŕōśś ţĥē Ķàńďēv ĩńţēŕƒàćē",
   "chooseWhatOpensWhenKandevStarts": "Ćĥōōśē ŵĥàţ ōƥēńś ŵĥēń Ķàńďēv śţàŕţś ōŕ ŷōũ vĩśĩţ ĩţś ƀàŕē ĥōḿē ƥàĝē",
   "chooseWhichTranscriptNavigationControls": "Ćĥōōśē ŵĥĩćĥ ţŕàńśćŕĩƥţ ńàvĩĝàţĩōń ćōńţŕōĺś àƥƥēàŕ, ĩńćĺũďĩńĝ ţĥē ďēśķţōƥ-ōńĺŷ śţĩćķŷ ƥŕōḿƥţ ƀàŕ",
+  "clickAUrlInTheTerminal": "Ćĺĩćķ à ŨŔĹ ĩń ţĥē ţēŕḿĩńàĺ ţō ōƥēń ĩţ.",
   "cmdCtrlEnterToSend": "Ćḿď/Ćţŕĺ+Ēńţēŕ ţō śēńď",
```
[the remark this answers is not in the document]
Comment (jcfs) [reply apps/web/src/locales/pseudo/settings.json:19]: Declining — this one can't be fixed in the file it's reported on.

`src/locales/pseudo/settings.json` is generated by `pnpm run i18n:pseudo`, and `i18n:check` gates `en`/`pseudo` sync, so a hand-edit is reverted by the next regeneration and would fail CI in between. Preserving these tokens has to happen in `scripts/generate-pseudo-locale.mjs`, which is i18n foundation code currently shared by three sibling migration PRs — changing it here would rewrite every pseudo catalog and collide with all of them.

Three of the six suggested lines (`cmdCtrlEnterToSend`, `pressCmdCtrlEnterToSend`, `pressEnterToSendMessagesPress`) are also pre-existing entries not in this PR's diff.

On the substance: `pseudo` is a QA-only locale built into dev/e2e only and never shipped. Its job is to make un-externalized English text stand out, not to be readable — and the `en` catalog users actually see keeps `Menlo / Monaco`, `px` and the keyboard names literal. Worth a separate PR against the generator if we want higher fidelity there.

File: apps/web/components/settings/terminal-settings.tsx [hunk 3]
```diff
@@ -26,16 +26,24 @@ import { useShellSettings } from "@/hooks/domains/settings/use-shell-settings";
...
-  icons: "Nerd Fonts",
-  ligatures: "Programming",
-  system: "System",
+/**
+ * Catalog KEYS, not copy. A module-scope table is evaluated once at import, so a
+ * `t()` here would freeze at the boot locale and never update on a switch — and
+ * neither the lint guard nor the pseudo-locale can see that. Resolved at render
+ * in `FontGroupOptions`. The record keys are `FontCategory` values (data) and
+ * stay untranslated.
+ */
```
[the remark this answers is not in the document]
Comment (jcfs) [reply apps/web/components/settings/terminal-settings.tsx]: The quoted rule isn't in this repo — `grep -rn "multi-paragraph docstrings\|multi-line comment blocks" --include="*.md" .` returns nothing, and the merged i18n foundation (#2097) uses exactly this comment shape for exactly this hazard in `startup-page-settings-card.tsx`, with `eslint.i18n.options.mjs` leaning on multi-line blocks throughout. Matching the neighbouring code is the convention I'm following.

The block was longer than it needed to be, though, so I trimmed it from six lines to three in dc6f9cb, keeping only the non-obvious part (freezes at boot locale; invisible to both checks).
